### ISSUE TITLE
False positive function detection

### ISSUE THREAD

**Issue by paztis** [arthurgeron/eslint-plugin-react-usememo#63]
For now any function that start with a capital letter is considered as a React FunctionComponent.
We have too much false positives on our side.

Don't you have a way to verify the arguments of the function for example ? only 1 argument ? This will avoid to detect this for example
```
export const Test = (arg1: string, arg2: string, arg3: string): string => {
    return `${arg1} ${arg2} ${arg3}`;
};
```
In case you have typescript signature you can verify the return type also (JSX.Element) ?
